src/libstd/sys/unix/ext/process.rs

```diff
@@ -25,6 +25,11 @@ pub trait CommandExt {
     #[stable(feature = "rust1", since = "1.0.0")]
     fn gid(&mut self, id: u32) -> &mut process::Command;
 
+    /// Sets the supplementary group IDs for the calling process. Translates to
+    /// a `setgroups` call in the child process.
+    #[stable(feature = "rust1", since = "1.0.0")]
```

Comment:
This cannot have existed since Rust 1.0...

Reply:
Oh whoops, I just copied it from `gid` and `uid`. I didn't really understood what it meant.